extensions/scripting/lua-bindings/manual/extension/axlua_extension_manual.cpp

```diff
@@ -48,7 +48,7 @@ class LuaRefMap : public Object
 
     ~LuaRefMap()
     {
-        AXLOGI("deallocing LuaRefMap: {}", this);
```

Comment:
Should be `AXLOGI("deallocing LuaRefMap: {}", fmt::ptr(this));`

Note: all non fmtlib style LOG macros shoud deprecated